core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -232,10 +262,12 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
 
       val unzipped = taskHeadersAndCssClasses.unzip
 
+      val currentTime = System.currentTimeMillis()
       val taskTable = UIUtils.listingTable(
         unzipped._1,
         taskRow(hasAccumulators, stageData.hasInput, stageData.hasOutput,
-          stageData.hasShuffleRead, stageData.hasShuffleWrite, stageData.hasBytesSpilled),
+          stageData.hasShuffleRead, stageData.hasShuffleWrite,
```

Comment:
style: each parameter should be on it's own line when they need to be line-wrapped like this